### DISCUSSION TITLE
@Mike777Khoo Mike，马上九月份了，还没有看到ios新版本

### DISCUSSION THREAD

**Discussion by peisy** [getlantern/lantern-forum-cn#810] [General]
一晃，都喊了快一年了。

**Comment by MaxMKMax** [comment]
@peisy 对的. 我也讲到无力了. 我只能说呢, 还是等iOS批准放人. 我感到深深的抱歉!!!

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/549cfe71-2511-40ea-8957-caef754bc63c)
说是说有进展的.
